en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-320.md

```diff
@@ -6,26 +6,4544 @@
+        
+        UPDATE REG_RESOURCE_PROPERTY SET REG_PATH_ID=(SELECT REG_PATH_ID FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_PROPERTY.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_RATING SET REG_PATH_ID=(SELECT REG_PATH_ID FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_RATING.REG_VERSION);
+        
+        -- Delete versioned tags, were the PATH_ID will be null for older versions --
+        delete from REG_RESOURCE_PROPERTY where REG_PATH_ID is NULL;
+        
+        delete from REG_RESOURCE_RATING where REG_PATH_ID is NULL;
+        
+        delete from REG_RESOURCE_TAG where REG_PATH_ID is NULL;
+        
+        delete from REG_RESOURCE_COMMENT where REG_PATH_ID is NULL;
+        
+        delete from REG_PROPERTY where REG_ID NOT IN (select REG_PROPERTY_ID from REG_RESOURCE_PROPERTY);
+        
+        delete from REG_TAG where REG_ID NOT IN (select REG_TAG_ID from REG_RESOURCE_TAG);
+        
+        delete from REG_COMMENT where REG_ID NOT IN (select REG_COMMENT_ID from REG_RESOURCE_COMMENT);
+        
+        delete from REG_RATING where REG_ID NOT IN (select REG_RATING_ID from REG_RESOURCE_RATING);
+        
+        -- Update the REG_PATH_NAME column mapped with the REG_RESOURCE table --
+        UPDATE REG_RESOURCE_TAG SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_TAG.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_PROPERTY SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_PROPERTY.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_COMMENT SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_COMMENT.REG_VERSION);
+        
+        UPDATE REG_RESOURCE_RATING SET REG_RESOURCE_NAME=(SELECT REG_NAME FROM REG_RESOURCE WHERE REG_RESOURCE.REG_VERSION=REG_RESOURCE_RATING.REG_VERSION);
+        ```
+
+!!! warning "Not recommended"
+    If you decide to proceed with registry resource versioning enabled, add the following configuration to the `<NEW_API-M_HOME>/repository/conf/deployment.toml` file of new WSO2 API Manager. 
+    
+    ```
+    [registry.static_configuration]
+    enable=true
+    ```
+    
+    !!! note "NOTE"
+        Changing these configuration should only be done before the initial API-M Server startup. If changes are done after the initial startup, the registry resource created previously will not be available.
+
+-   [Step 1 - Migrate the API Manager configurations](#step-1-migrate-the-api-manager-configurations)
+-   [Step 2 - Upgrade API Manager to 3.2.0](#step-2-upgrade-api-manager-to-320)
+-   [Step 3 - Optionally, migrate the configurations for WSO2 API-M Analytics](#step-3-optionally-migrate-the-configurations-for-wso2-api-m-analytics)
+-   [Step 4 - Restart the WSO2 API-M 3.2.0 server](#step-4-restart-the-wso2-api-m-320-server)
```

Comment:
Steps 3 and 4 don't appear in this page, but they are listed in the page TOC

Reply:
removed steps 3 and 4 in TOC with https://github.com/wso2/docs-apim/pull/1775/commits/6f9f104be57b16bafee6c987a395bc37f94d2cc5